src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10690,6 +10786,30 @@ definitions:
         $ref: '#/definitions/PermissionsInfo'
       definition:
         $ref: '#/definitions/Definition'
+  calculateSizeResponse:
+    type: object
+    properties:
+      id:
+        type: string
+        description: Provides the id for which POST/calculateSize has been initiated.
+      size:
+        type: string
+        description: Provides a folder size in bytes.
+      calculatedAtTime:
+        type: string
+        description: Provides the time when the calculating folder size will be completed.
+      numberOfFiles:
+        type: integer
+        description: Provides the number of files in the folder.
+      status:
+        type: string
+        description: Provides a status that signifies the completion of the calculateSize mechanism.
+  sizeAcknowledgedResponse:
+    type: object
+    properties:
+      nodeId:
+        type: string
+        description: Provides nodeId indicating that the request has been acknowledged.
```

Comment:
I'd expect the response to contain the id of the background job.

Reply:
We provide the **nodeId** in the POST Endpoint response instead **backgroundJobId** because it's removed once the action is completed. As a result, we cannot discover the size in the GET Endpoint response.

Reply:
Now, we are considering a different execution ID instead of a node ID.